py3status/modules/whoami.py

```diff
@@ -21,17 +19,26 @@ class Py3status:
     """
     """
     # available configuration parameters
-    cache_timeout = 1800
     format = '{username}'
 
+    class Meta:
+        deprecated = {
+            'remove': [
+                {
+                    'param': 'cache_timeout',
+                    'msg': 'obsolete parameter remove',
```

Comment:
sorry I'm not clear, I mean the message will not be very clear/proper because it will basically show up as

`DEPRECATION WARNING: whoami `cache_timeout` obsolete parameter remove`